README.md

```diff
@@ -41,3 +41,23 @@ n = 10**5
 sleep_durations = [random.uniform(0, 5.0) for _ in range(n)]
 asyncio.run(run_multiple_sleeps(sleep_durations=sleep_durations))
 ```
+
+## Demo
+A complete demo of `tqdm_publisher` can be found in the `demo` directory, which shows how to forward progress updates from the same `TQDMPublisher` instance to multiple clients.
+
+To run the demo, first install the dependencies:
+```bash
+pip install -r demo/requirements.txt
+```
+
+Then, run the Python script for the demo:
+```bash
+python demo/main.py
```

Comment:
The point of exposing the entrypoint in the TOML is so that the key in the TOML should become registered as a global command

That is, any Python/conda environment with this package installed, you should be able to run

```
tqdm_publisher-demo-server
```

from the terminal and it should do the same thing as calling python on the file (main difference being, you don't need to know where that file actually lives)